regparser/api_writer.py

```diff
@@ -128,6 +128,7 @@ class Client:
 
     def __init__(self):
         if settings.API_BASE:
+            print 'writing to {0}'.format(settings.API_BASE)
```

Comment:
Here's a print statement that either needs to be a logging statement, or needs to be removed.